Lobe_iOS/Models/PredictionLayer.swift

```diff
@@ -72,6 +72,11 @@ class PredictionLayer: NSObject {
       }
       onComplete(request)
     })
+    
+    /// Set center cropping for the expected image size.
```

Comment:
@mbeissinger comment code here for your review.

Reply:
yep looks good